subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -789,7 +926,15 @@ static int iso_handle(const struct device *dev, struct net_buf *buf)
 	err = hci_iso_handle(buf, &evt);
 	if (evt) {
 		LOG_DBG("Replying with event of %u bytes", evt->len);
+
+#if defined(CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE)
 		bt_recv_prio(dev, evt);
+
+#else /* CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE */
+		struct hci_driver_data *data = dev->data;
+
+		data->recv(dev, evt);
+#endif /* CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE */
```

Comment:
We could consider a helper function for these lines as they are copied several times in this file